apps/src/templates/instructions/InlineFeedback.jsx

```diff
@@ -11,7 +11,10 @@ class InlineFeedback extends Component {
     borderColor: PropTypes.string,
     extra: PropTypes.string,
     message: PropTypes.string.isRequired,
-    styles: PropTypes.object
+    styles: PropTypes.object,
+    isMinecraft: PropTypes.bool,
+    skinId: PropTypes.string,
+    textToSpeechEnabled: PropTypes.string
```

Comment:
@bethanyaconnor i assume this was a typo? we're passing in a bool and `ChatBubble` is expecting a bool, so i'm getting a "Failed prop type" warning from this. i'm going to fix it unless i'm misunderstanding somerthing.

Reply:
Yes, but it should've been fixed in https://github.com/code-dot-org/code-dot-org/pull/39618. Is it still happening?

Reply:
ah my bad, i haven't rebased onto staging in a while. thanks!!